src/PostgresOutbox/Table/EventsAppender.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections;
+using Npgsql;
+using PostgresOutbox.Serialization;
+
+namespace PostgresOutbox.Table;
+
+public static class EventsAppender
```

Comment:
**SUGGESTION:** Might not be for this PR, but I think that it'd be better to rename `Event` into `Message`, I know that's my mishap made initially, I can handle that in post-merge, if you don't have time to rename it :)

Reply:
Definitely